.github/workflows/IWYU_scan.yml

```diff
@@ -0,0 +1,108 @@
+name: IWYU_scan
+
+concurrency:
+  group: build-${{ github.head_ref }}
```

Comment:
`IWYU_scan-${{ github.head_ref }}`. This group should be an unique value. In your case, it will cancel the `build.yml`.

Reply:
Ok, so this is why the build was failing

Reply:
Fixed it